commit 5983b506e9a692a00987d86a60390028d3ad73ce
Author: fryshorts <leonhard@in-verted.de>
Date:   Sun May 3 17:01:09 2015 +0200

    linux-pulseaudio: Add include directory to cmake
    
    Add the include directories found by cmake to the pulseaudio plugin.
    This allows for the plugin to compile when the pulseaudio headers were
    found in a directory that is not normally in the search path of the
    compiler (e.g. /usr/local/include)

--- plugins/linux-pulseaudio/CMakeLists.txt
+++ plugins/linux-pulseaudio/CMakeLists.txt
@@ -13,7 +13,10 @@ elseif(NOT PULSEAUDIO_FOUND)
 	return()
 endif()
 
-include_directories(SYSTEM "${CMAKE_SOURCE_DIR}/libobs")
+include_directories(
+	SYSTEM "${CMAKE_SOURCE_DIR}/libobs"
+	${PULSEAUDIO_INCLUDE_DIR}
+)
 
 set(linux-pulseaudio_SOURCES
 	linux-pulseaudio.c
